capstone/src/main/webapp/WEB-INF/web.xml

```diff
@@ -11,5 +11,6 @@
     <filter-mapping>
         <filter-name>authFilter</filter-name>
         <url-pattern>/dashboard</url-pattern>
+        <url-pattern>/forms</url-pattern>
```

Comment:
Apparently /form is a reserved URL. So I changed the mapping of all of the /form to /forms.

Reply:
Interesting 🤔 I did not know that